easy_phi/options.conf

```diff
@@ -0,0 +1,4 @@
+VERSION = "0.1"
```

Comment:
This file is not part of app code and should be installed to /etc/easy-phi.conf
It is achieved by `script` option in setup.py. In our code it means that it should be placed into scripts folder